application/frontend/web/index.php

```diff
@@ -1,16 +1,5 @@
 <?php
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
 
-require(__DIR__ . '/../../vendor/autoload.php');
-require(__DIR__ . '/../../vendor/yiisoft/yii2/Yii.php');
-require(__DIR__ . '/../../common/config/bootstrap.php');
-require(__DIR__ . '/../config/bootstrap.php');
+$application = require('load-app.php');
```

Comment:
I don't want to have to worry about keeping those files in sync.  I refactored this a bit to eliminate that visibility issue and deal with whether to `run()` or not to the initiator.  https://github.com/silinternational/idp-id-broker/commit/f6f89e36cd91783e90e823199621b2fdc2ae0edb